test/e2e/atlas/helper_test.go

```diff
@@ -720,6 +720,71 @@ func deleteAllPrivateEndpoints(t *testing.T, cliPath, projectID, provider string
 	require.True(t, done, "failed to clean all private endpoints")
 }
 
+func deleteAllStreams(t *testing.T, cliPath, projectID string) {
+	t.Helper()
+
+	streams := listStreamsByProject(t, cliPath, projectID)
+	if streams.TotalCount != nil && *streams.TotalCount == 0 {
```

Comment:
```suggestion
	if streams.GetTotalCount() == 0 {
```
https://github.com/mongodb/atlas-sdk-go/blob/main/docs/doc_5_best_practices.md#using-getters-instead-of-direct-field-access

Reply:
thanks! updated